examples/relativistic/constant_force_trajectory.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+from sympy import symbols, dsolve, solve, Eq
+from symplyphysics import print_expression
+from symplyphysics.laws.dynamics import (
+    force_is_derivative_of_momentum as newtons_second_law,
+)
+from symplyphysics.laws.relativistic import (
+    relativistic_momentum as relativistic_momentum_law,
+)
+
+# Description
+## A particle with an initial momentum `p_0` starts moving at a moment `t = 0` due to a constant
+## force `F`. Find the time dependence of the particle's velocity and the distance covered.
+
+# Momentum is expressible as a function of time
+
+force, initial_momentum = symbols("force, initial_momentum", positive=True)
+
+time = newtons_second_law.time
+momentum_function = newtons_second_law.momentum
+
+momentum_via_time = dsolve(
+    newtons_second_law.law.subs(newtons_second_law.force(time), force),
+    momentum_function(time),
+    ics={momentum_function(0): initial_momentum},
+).rhs
+
+# We can find velocity as a function of momentum
+
+speed, rest_mass, momentum = symbols("speed, rest_mass, momentum", positive=True)
+
+relativistic_momentum_eqn = relativistic_momentum_law.law.subs({
+    relativistic_momentum_law.momentum: momentum,
+    relativistic_momentum_law.velocity: speed,
+    relativistic_momentum_law.rest_mass: rest_mass,
+})
+
+speed_via_momentum = solve(relativistic_momentum_eqn, speed)[0]
+
+# Now we can express speed as a function of time
+
+speed_via_time = speed_via_momentum.subs(momentum, momentum_via_time)
+
+print("Formula of particle speed as a function of time:\n")
+print(print_expression(Eq(speed, speed_via_time)))
+
+# Integrate the velocity formula over time to obtain the distance formula.
+# Note that the distance covered is zero at `t = 0`.
+
+distance, length_term = symbols("distance, length_term", positive=True)
+
+distance_via_time = speed_via_time.integrate(time)
+
+length_term_expr = distance_via_time.subs(time, 0)
```

Comment:
But distance at zero time is zero. Why you do not use this condition?

Reply:
I renamed the symbols so that they're less confusing.

Reply:
Still, this integration constant should be zero in our example.

Reply:
Why? When we integrate speed over time we do not get distance immediately. We obtain some function of time that is not necessarily zero at `t = 0`.

Reply:
You even have a note about it:
`Note that the distance covered is zero at t = 0`

Reply:
Yes, but why not simply equate it to zero and get the same distance equation?

Reply:
It's weird that SymPy `integrate()` does not add integration constant to the result.
For your example it should be `x(t) = -cos(t) + C1` after integration, where `C1` is initial position.

Reply:
From SymPy documentation:
```
Note that SymPy does not include the constant of integration. If you want it, you can add one yourself, or rephrase your problem as a differential equation and use dsolve to solve it, which does add the constant (see [Solving Differential Equations](https://docs.sympy.org/latest/tutorials/intro-tutorial/solvers.html#tutorial-dsolve)).
```
So you effectively wrote a workaround for this issue, by subtracting constant term from the result.